receiver/vcenterreceiver/metadata.yaml

```diff
@@ -421,6 +443,51 @@ metrics:
       value_type: double
     attributes: [throughput_direction, object_name]
     extended_documentation: As measured over the most recent 20s interval.
+  vcenter.host.vsan.throughput:
+    enabled: false
+    description: The vSAN throughput of a host.
+    unit: "By/s"
+    gauge:
+      value_type: double
+    attributes: [vsan_throughput_direction]
+    warnings:
+      if_enabled_not_set: "this metric will be enabled by default starting in release v0.106.0"
+  vcenter.host.vsan.operations:
+    enabled: false
+    description: The vSAN IOPs of a host.
+    unit: "{operations/sec}"
+    gauge:
+      value_type: int
+    attributes: [vsan_operation_type]
+    warnings:
+      if_enabled_not_set: "this metric will be enabled by default starting in release v0.106.0"
+  vcenter.host.vsan.latency.avg:
+    enabled: false
+    description: The host latency while accessing VSAN storage.
+    unit: "us"
+    gauge:
+      value_type: int
+    attributes: [vsan_latency_type]
+    warnings:
+      if_enabled_not_set: "this metric will be enabled by default starting in release v0.106.0"
+  vcenter.host.vsan.congestion:
+    enabled: false
+    description: The congestions of IOs generated by all vSAN clients in the host.
+    unit: "{congestions}"
```

Comment:
because this isn't given as a counter i think we will want to convert this to `congestions/sec`

Reply:
Can you explain this to me? I thought the congestions were a counter?

Reply:
Let me rephrase. Because it isn't given as a sum counter (from the start of time) vs a delta counter (over a somewhat arbitrary interval).

Reply:
resolve